enclave/src/main/java/org/hyperledger/besu/enclave/RequestTransmitter.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.enclave;
+
+public interface RequestTransmitter {
+
+  @FunctionalInterface
+  interface ResponseBodyHandler<T> {
+    T convertResponse(final int statusCode, final byte[] body);
+  }
+
+  <T> T post(
+      String mediaType,
+      String content,
+      String endpoint,
+      ResponseBodyHandler<T> responseBodyHandler);
+
+  <T> T get(
+      String mediaType,
+      String content,
```

Comment:
No need for content field on the get request

Reply:
done